### PR TITLE
XRDDEV-1394 API keys have only observer role on a secondary node

### PR DESCRIPTION
**Pull request by petkivim**
- Remove from API keys all non-observer roles on a secondary node.
- Load node.ini in the xroad-proxy-ui-api process.
- Add a note about the API keys in the Security Server LB installation guide.

JIRA issue: https://jira.niis.org/browse/XRDDEV-1394

### CODE REVIEW

**Review by carohauta — APPROVED** [review]
🦄

**Comment by carohauta** [comment]
Looks good 👍 Just a couple of conflicts remain

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/DiagnosticService.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ private JsonObject sendGetRequest(String address) throws DiagnosticRequestExcept
 
         try (CloseableHttpClient httpClient = HttpClientBuilder.create().setDefaultRequestConfig(config).build();
              CloseableHttpResponse response = httpClient.execute(request)) {
-            HttpEntity resEntity = response.getEntity();
+               HttpEntity resEntity = response.getEntity();
```
Comment (carohauta) [inline src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/DiagnosticService.java:144]:
```suggestion
            HttpEntity resEntity = response.getEntity();
```

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/DiagnosticService.java [hunk 2]
```diff
@@ -141,7 +141,7 @@ private JsonObject sendGetRequest(String address) throws DiagnosticRequestExcept
 
         try (CloseableHttpClient httpClient = HttpClientBuilder.create().setDefaultRequestConfig(config).build();
              CloseableHttpResponse response = httpClient.execute(request)) {
```
Comment (carohauta) [inline src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/DiagnosticService.java:143]:
```suggestion
                CloseableHttpResponse response = httpClient.execute(request)) {
```

**Review by carohauta — COMMENTED** [review]
Well actually two indentation errors were found

**Comment by petkivim** [comment]
Resolved merge conflicts.

**Pull request merged**